Contribute/how-to-write-links.md

```diff
@@ -79,41 +81,53 @@ You do not have to create anchors. They're automatically generated at publishing
 
 Because include files are located in another directory, you must use longer relative paths. To link to an article from an include file, use this format:
 
-    [link text](../articles/folder/article-name.md)
+   ```markdown
+   [link text](../articles/folder/article-name.md)
+   ```
 
 ## Links in selectors
 
-If you have selectors that are embedded in an include--as does the Azure documentation team--use the following link structure:
+A selector is a navigation component that appears in a docs article as a drop down list. When a reader selects a value in the drop down, the browser opens the selected article. Typically the selector list contains links to closely related articles, for example the same subject matter in multiple programming languages or a closely related series of articles. 
```

Comment:
@JasonWHowell It's been a long while since I reviewed this topic, but I think the edit adds clarity. 
@gewarren A small style-related edit request: Please use "drop-down list". (Hyphenate a modifier, per the Microsoft Writing Style Guide, [https://docs.microsoft.com/en-us/style-guide/punctuation/dashes-hyphens/hyphens](https://docs.microsoft.com/en-us/style-guide/punctuation/dashes-hyphens/hyphens)

Reply:
@TerryGLee That was Jason's edit, not mine.

Reply:
@TerryGLee Fixed. Thanks!

Reply:
@TerryGLee My biggest doubt is that I can't see anywhere in docs where we use this code syntax with dual drop-downs. Have we tested that kind of drop-down? `[AZURE.SELECTOR-LIST (Dropdown1 | Dropdown2 )]`  

Should I use the other kind of drop down instead?
`[!div class="op_single_selector"]`

Reply:
@JasonWHowell Unfortunately, I am not the expert on this. My role in the topic was a "lift & shift" by collating content that was scattered everywhere and coalescing it into one document. (At the time, most of the internal guidance was written by Azure folks, and part of my role was to help make it a one-size-fits-all across the board.) That being said, what functionality is available and the intent behind it might best be pursued with the experts... Perhaps this is an issue that would be best pursued in Teams?